include/swift/DependencyScan/StringUtils.h

```diff
@@ -13,6 +13,7 @@
 #include "swift-c/CommonString/CommonString.h"
 #include "llvm/ADT/StringRef.h"
 #include "llvm/ADT/StringSet.h"
+#include "llvm/ADT/SmallVector.h"
```

Comment:
This is not needed for a header file that is not changed.

Reply:
Removed.